src/AbstractContainer.php

```diff
@@ -95,10 +92,8 @@ protected function sort()
 
     /**
      * Notifies container that the order of pages are updated
-     *
-     * @return void
      */
-    public function notifyOrderUpdated()
+    public function notifyOrderUpdated(): void
```

Comment:
The same applies here: if a user has a custom container implementation, this would break the application.